drivers/google/option.go

```diff
@@ -134,9 +134,16 @@ func WithUserDataKey(text string) Option {
 }
 
 // WithZone returns an option to set the target zone.
-func WithZone(zone string) Option {
+func WithZones(zones []string) Option {
```

Comment:
one minor change for consistency ... can you do `WithZones(zones ...string)` similar to this:
https://github.com/drone/autoscaler/blob/350f5a87a679c5797f9d35a1b448362843cdcf5f/drivers/google/option.go#L96

Reply:
Got it, thanks 😄